lib/api/web-element/commands/isSelected.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Determines if an element is selected.
+ *
+ * For more info on working with DOM elements in Nightwatch, refer to the <a href="https://nightwatchjs.org/guide/writing-tests/finding-interacting-with-dom-elements.html">Finding & interacting with DOM Elements</a> guide page.
+ *
+ * @example
+ * export default {
+ *   demoTest(browser: NightwatchAPI): void {
+ *     const result = browser.element('#main ul li a.first').isSelected();
+ *       .assert.valueEquals(true);
```

Comment:
The last line above does not make any sense. Can you try running the test by yourself first before putting it in the JSDoc? Also, it'd be better if we use the BDD (`describe`, `it`) style of writing tests instead of `export default`.

Reply:
I have added a BDD testcase (have tested it and it works) for the JSDoc. Will this work?
added in [345b572](https://github.com/nightwatchjs/nightwatch/pull/4038/commits/345b572409c74688b489e1a208f3bfb2a75152cb)

```
describe('isSelected Demo', function() {
  before(browser => {
    browser
      .navigateTo('https://www.ecosia.org/');
  });

  it('Demo test ecosia.org', function(browser) {
    browser
      .waitForElementVisible('body')
      .element('.search-form__input-wrapper')
      .isSelected(function callback(result) {
        assert.strictEqual(result.value, true);
      });
  });

  after(browser => browser.end());
});
```

Reply:
Looks good, thanks! Have posted a few suggestions though.